SettingPresets/Products/app-extension.messages.yml

```diff
@@ -0,0 +1,2 @@
+ASSETCATALOG_COMPILER_APPICON_NAME: iMessage App Icon
+LD_RUNPATH_SEARCH_PATHS: "$(inherited) @executable_path/Frameworks @executable_path/../../Frameworks"
```

Comment:
> @executable_path/../../Frameworks

Wow, this is new👀

Reply:
What’s new?

Reply:
I mean the location path was new to me.  NVM👋